dynamic_rest/renderers.py

```diff
@@ -91,6 +91,9 @@ def process(result):
             c for c in context['columns'] if c not in self.COLUMN_BLACKLIST
         ]
         context['details'] = context['columns']
+        context['error_form'] = context['error_form']
+        if context['error_form']:
```

Comment:
I think this would break error-handling for single-record updates (edit/create one instance). To get around that, check for the presence of the "errors" key:

```
bulk_errors = context['response'].data.get('errors')
if context['error_form'] and bulk_errors:
   # this is a bulk upload error
   context['errors'] = bulk_errors
```